pkg/cloudprovider/aws/apis/v1alpha1/register.go

```diff
@@ -81,7 +81,7 @@ var (
 func init() {
 	Scheme.AddKnownTypes(schema.GroupVersion{Group: v1alpha5.ExtensionsGroup, Version: "v1alpha1"}, &AWS{})
 	v1alpha5.RestrictedLabelDomains = v1alpha5.RestrictedLabelDomains.Insert(RestrictedLabelDomains...)
-	v1alpha5.WellKnownLabels = v1alpha5.WellKnownLabels.Insert(
```

Comment:
NOTE: `sets.Insert()` updates the receiver itself, so no need to assign.
https://github.com/kubernetes/apimachinery/blob/f916759cb6b8547418dc7708876ecab5c1961448/pkg/util/sets/string.go#L48-L54